src/printer/components/dataFormat.js

```diff
@@ -54,16 +54,16 @@ function formatEstimatedTime(time) {
 
     let plus_days = "";
     if (end.getDate() == now.getDate() && end.getMonth() == now.getMonth()) {
-      plus_days = "today at ";
+      plus_days = translate("prop.today-at") + " ";
     } else if (
       end.getDate() == tomorrow.getDate() &&
       end.getMonth() == tomorrow.getMonth()
     ) {
-      plus_days = "tomorrow at ";
+      plus_days = translate("prop.tmw-at") + " ";
     } else {
       let options = { month: "numeric", day: "numeric" };
       const final_date = end.toLocaleString(window.navigator.language, options);
-      plus_days = `${final_date} at `;
+      plus_days = `${final_date} ${translate("prop.at")} `;
```

Comment:
Have you tested this? If there will be some issue with missing translation, just assign a variable first
```
const at = translate("prop.at");
plus_days = `${final_date} ${at} `;
```

Reply:
indeed this worked on my computer

Reply:
I'll change for your suggestion
Thank you